packages/components/src/promo-card/index.tsx

```diff
@@ -1,29 +1,55 @@
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
 import './style.scss';
-import { AppStoreSVG, JetpackSVG, QRCodeSVG, WordPressSVG } from './svg-icons';
+import { AppStoreSVG, QRCodeSVG, WordPressJetpackSVG } from './svg-icons';
 
 export type PromoCardProps = {
-	data: {
-		title: string | null;
-		message: string | null;
-		cta?: string;
-	};
+	className?: string;
+	isMobile?: boolean;
 };
 
-export default function PromoCard( { data: { title, message, cta } }: PromoCardProps ) {
-	const isAppStore = cta === 'app-store';
+export default function PromoCard( { className, isMobile }: PromoCardProps ) {
+	// Using useTranslate() with interpolation to set up the title/message.
+	// https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md
+	const translate = useTranslate();
+	const components = {
+		a: <a href="https://wp.com/apps" />,
+	};
+	const getTitle = () => {
+		return translate( 'Bring your stats with you using the {{a}}Jetpack{{/a}} mobile app', {
+			components,
+		} );
+	};
+	const getMessage = () => {
+		let string;
+		if ( isMobile ) {
+			string = translate(
+				'Check your stats on-the-go and get real-time notifications with the Jetpack mobile app.'
+			);
+		} else {
+			string = translate(
+				'Visit {{a}}wp.com/app{{/a}} or scan the QR code to download the Jetpack mobile app.',
+				{ components }
+			);
+		}
+		return string;
+	};
```

Comment:
Could we inline this within the component tree? Unless we're planning on using these values multiple times in the tree, there's no need to declare them as separate variables outside of the returned tree.

Reply:
Addressed in #70023.